ts/@live-compositor/core/package.json

```
@@ -28,12 +28,14 @@
     "/cjs"
   ],
   "devDependencies": {
+    "@types/react": "^18.3.3",
     "@types/react-reconciler": "0.28.8"
   },
   "dependencies": {
     "react-reconciler": "0.29.2"
   },
   "peerDependencies": {
-    "live-compositor": "^0.1.0"
+    "react": "^18.3.1",
```

```suggestion
    "react": "*",
```
we need to have a large range, if I remember correctly more than one react in app will not work